### DISCUSSION TITLE
Program_03_2

### DISCUSSION THREAD

**Discussion by LucarioKid** [gberl001/SP24-ENR261-180#2] [Q&A]
For part 3 of program_03_2, when using the shown equation, my results did not match those shown in the book or on the github page. However, when I double checked my math by hand, the result matched the one I reached that did not match the one in the book. I would like to know if this is intentional or not, and if anyone else has had this experience.

**Comment by gberl001** [comment]
The values in the example output should match your values, if they don't then something is off somewhere. If you need more help, post a code snippet of one of the parts and I can take a look, or let me know in class.

**Reply by LucarioKid** [reply]
<img width="807" alt="Screen Shot 2024-02-09 at 4 10 31 PM" src="https://github.com/gberl001/SP24-ENR261-180/assets/91693995/659055e4-868f-4dd1-bc5a-2f5bc057d263">

**Reply by gberl001** [reply]
It looks like you made a slight mixup using addition instead of multiplication for the denominator.

`element^2 + (element+2)^2` instead of `element^2 * (element+2)^2`
